orval.config.ts

```diff
@@ -0,0 +1,31 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
This config file should be moved into `src/core/material-list-api` as every generated client needs to have there own config file. Then run orval with `--config` alternative rename the file to `material-list-api.orval.config.ts`?

Reply:
Forget about this one @kasperg has just made me adware that it is possible to have more than one client defined in the same config file :-)